### PR TITLE
Some cleanup on the ensembldb code

### PR DESCRIPTION
**Pull request by gamazeps**
- Remove the hardcoded formating for the db path
- Cleanup error messages
- Improve `__repr__`

### CODE REVIEW

File: src/genomic_features/ensembl/ensembldb.py [hunk 1]
```diff
@@ -55,8 +53,10 @@ def annotation(
     >>> gf.ensembl.annotation("Hsapiens", "108")
     """
     try:
-        sqlite_file_path = retrieve_annotation(
-            ENSEMBL_URL_TEMPLATE.format(species=species, version=version)
+        sqlite_file_path = retrieve_annotation(os.path.join(
+                    BIOC_ANNOTATION_HUB_URL,
+                    f'AHEnsDbs/v{version}/EnsDb.{species}.v{version}.sqlite'
+            )
```
[the remark this answers is not in the document]
Comment (gamazeps) [reply src/genomic_features/ensembl/ensembldb.py]: Done with an f-string as you recommended

**Comment by gamazeps** [comment]
All comments are addressed, PTAL

File: src/genomic_features/ensembl/ensembldb.py [hunk 2]
```diff
@@ -3,6 +3,7 @@
 import warnings
 from functools import cached_property
 from itertools import product
+import os
```
Comment (ivirshup) [inline src/genomic_features/ensembl/ensembldb.py:6]:
```suggestion
```

Made me realize pre-commit isn't running on PRs

**Review by ivirshup — APPROVED** [review]
lgtm, thanks!

**Pull request merged**